desktop-app/app/components/Addressinput/index.js

```diff
@@ -39,6 +39,7 @@ class AddressBar extends React.Component<Props> {
       previousAddress: props.address,
       finalUrlResult: null,
       canShowSuggestions: false,
+      cursor: 0,
```

Comment:
@abakermi I found a small bug, we need to reset the cursor to `0` once the user loads an address. 

Right now the cursor is not reset and persisted across suggestions.

Reply:
you mean reset the cursor within handle input change, just when user start typing 
```
  _handleInputChange = e => {
    this.setState(
      {userTypedAddress: e.target.value, canShowSuggestions: true,
      cursor:0},  // here
      () => {
        this._filterExistingUrl();
      }
    );
  };
```

Reply:
what anout the color, I think its nice and consistent like that 

![Peek 2020-07-24 14-46](https://user-images.githubusercontent.com/60294727/88388107-878e0a00-cdbc-11ea-8e19-86ea1a05c96f.gif)

Reply:
Yeah, this one looks much better, let's keep it Thanks for suggesting it. 👍

Reply:
> you mean reset the cursor within handle input change, just when user start typing
@abakermi But this will reset it for every character typed right? That might be odd. 

We can reset it at the handler where we call the `address change` redux action.